src/Repository/PermissionRepo.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Laravolt\Acl\Repository;
+
+class PermissionRepo
+{
+    protected $model;
+
+    public function __construct()
+    {
+        $this->model = config('laravolt.acl.models.permission');
```

Comment:
Coba dibikin `$this->model = app(config('...')); Jadinya $this->model tipenya beneran sudah Eloquent object